packages/lib/server-only/user/generate-confirmation-token.ts

```diff
@@ -0,0 +1,44 @@
+import crypto from 'crypto';
+
+import { prisma } from '@documenso/prisma';
+
+import { ONE_HOUR } from '../../constants/time';
+import { sendConfirmationEmail } from '../auth/send-confirmation-email';
+
+const IDENTIFIER = 'confirmation-email';
+
+export const generateConfirmationToken = async ({ email }: { email: string }) => {
```

Comment:
`generateAndSendConfirmationToken` 🤔 

Or `sendConfirmationEmail` is also fine, we don't need to expose the inner workings here I think.